<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>731</width>
     <height>400</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leX"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leY"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sum = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSum">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSum">
        <property name="text">
         <string>Sum</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Lib path = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leLib"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnUnload">
        <property name="text">
         <string>Unload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Domain_ID = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDomainID"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetDomain">
        <property name="text">
         <string>setDomain</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>New Partition = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePartition"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetPartition">
        <property name="text">
         <string>setPubPartition</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>New Partitions = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePartitions"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddPubPartitions">
        <property name="text">
         <string>addPubPartitions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSubPartitions">
        <property name="text">
         <string>addSubPartitions</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnPub">
        <property name="text">
         <string>pub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopPub">
        <property name="text">
         <string>stopPub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSub">
        <property name="text">
         <string>sub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopSub">
        <property name="text">
         <string>stopSub</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopThread">
        <property name="text">
         <string>stopThread</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QPushButton" name="btnBatterystatus">
        <property name="text">
         <string>Power State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>OpenDB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>CreateDocs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>GetDocs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>ExportJSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>ImportJSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRepStart">
        <property name="text">
         <string>Rep Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRepStop">
        <property name="text">
         <string>Rep Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
